dev-infrastructure/templates/svc-infra.bicep

```diff
@@ -116,5 +123,34 @@ module serviceKeyVaultDevopsSecretsOfficer '../modules/keyvault/keyvault-secret-
   ]
 }
 
+// 
+//   N E T W O R K    S E C U R I T Y    P E R I M E T E R
+//
+
+module svcNSP '../modules/network/nsp.bicep' = {
+  name: 'nsp-svc-kv-${uniqueString(resourceGroup().name)}'
+  params: {
+    nspName: svcKVNSPName
+    location: serviceKeyVaultLocation
+  }
+}
+
+module svcNSPProfile '../modules/network/nsp-profile.bicep' = {
+  name: 'profile-svc-kv-${uniqueString(resourceGroup().name)}'
+  params: {
+    accessMode: svcKVNSPAccessMode
+    nspName: svcKVNSPName
+    profileName: svcKVNSPName
+    location: serviceKeyVaultLocation
+    associatedResources: [
+      serviceKeyVault.outputs.kvId
```

Comment:
we should not manage the same resource (the profile) in two places. i suggest we move the svc kv association to service-cluster.bicep

Reply:
It's a different NSP in the service key vault rg (global)

Reply:
ah i see. it is just in the global RG in DEV for reuse, in MSFT envs it is in the SVC RG. is there a way to get it into to NSP in the SVC RG or is there something limiting us from doing it?

Reply:
Hm, it's a trade off, cause we have it in the global in integrated dev and in the svc rg in int/stage. I tend to let it live near the KV, cause otherwise the template resources would be split over two RGs in integrated dev otherwise.

Reply:
i'm not sure i understand. isn't this "just" adding the SVC KV resourceID to the SVC NSP in svc-cluster.bicep?

Reply:
Please check lines above: https://github.com/Azure/ARO-HCP/pull/1698/files/64fbe064c7ce82a4e0edefb2545853e5f87e887e#diff-3af1c68f034209b2f567963719477f3483c0a6cf97a24497fe79efa81683c439R130-R136